stan/math/fwd/scal/fun/beta.hpp

```diff
@@ -50,8 +50,9 @@ template <typename T>
 inline fvar<T> beta(const fvar<T>& x1, const fvar<T>& x2) {
   const T beta_ab = beta(x1.val_, x2.val_);
   return fvar<T>(beta_ab,
-                 beta_ab * (x1.d_ * digamma(x1.val_) + x2.d_ * digamma(x2.val_)
```

Comment:
[comment]
It's much easier to review PRs if they don't change whitespace across lines, which shows up as a diff even with the hide whitespace settings on.